File: CompilationTest/FileTest/results/FileTest.java
import java.nio.file.Files;
import java.nio.file.Paths;

class Program {
    public static void main(String[] args) throws Exception {
        Files.write(Paths.get("test.txt"), "example content".getBytes());;
        String fileContent = new String(Files.readAllBytes(Paths.get("test.txt")));;
        System.out.println(fileContent);
    }
}

File: CompilationTest/FileTest/results/FileTest.cpp
#include <OneLang-File-v0.1/OneFile.hpp>
#include <iostream>

int main()
{
    OneFile::writeText(std::string("test.txt"), std::string("example content"));
    auto file_content = OneFile::readText(std::string("test.txt"));
    std::cout << file_content << std::endl;
    return 0;
}